### ISSUE TITLE
"Save as Image" crops and mangles the image

### ISSUE THREAD

**Issue by statsandthings** [rstudio/rstudio#3784]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.1.463 
    OS Version      : Windows 7 Professional
    R Version       : 3.5.1

### Steps to reproduce the problem
Run the following code

```
library(networkD3)
URL <- paste0(
  "https://cdn.rawgit.com/christophergandrud/networkD3/",
  "master/JSONdata/energy.json"
  )
Energy <- jsonlite::fromJSON(URL)
sankeyNetwork(Links = Energy$links, Nodes = Energy$nodes, Source = "source",
              Target = "target", Value = "value", NodeID = "name",
              units = "TWh", fontSize = 12, nodeWidth = 30)
```

then on the `Viewer` tab select either `Save as Image` or `Copy to Clipboard` and choose a width of 1,000 and a height of 618. Proceed to save and/or copy the plot and notice that it's cropped all weird. Here's what it looks like. Seems like a bug.

[![capture.png](https://i.postimg.cc/sftF7vjs/capture.png)](https://postimg.cc/K4rsyc3H)

**Comment by kevinushey** [comment]
I believe this issue is resolved in the [preview release](https://www.rstudio.com/products/rstudio/download/preview/) -- can you give it a download and let us know if the situation is indeed improved?

**Comment by statsandthings** [comment]
I can test the preview release. Just not on my production machine. Give me a few days and I'll be back. Thanks

**Comment by jmcphers** [comment]
Closing since it's solved in the 1.2 release. Thanks for following up, @statsandthings!
